components/dashboard/src/teams/Members.tsx

```diff
@@ -188,7 +211,16 @@ export default function() {
                     </Item>)}
             </ItemsList>
         </div>
-        {genericInvite && showInviteModal && <Modal visible={true} onClose={() => setShowInviteModal(false)}>
+        {genericInvite && showInviteModal && <Modal visible={true} onEnter={() => {
```

Comment:
**thought**: Instead of tracking this shall we track the [Copy Invite URL](https://github.com/gitpod-io/gitpod/pull/5168/files#diff-53ff0bdfe65ca0d0e6a4af4ac305ab6a4f94c852f14e509b915659e8d653c820R229) below?